[subgraph] various subgraph mapping fixes

kasparkallas opened the pull request:
- Fix `adjustmentFlowRate` on Pool
- Improve `poolTotalAmountReceivedUntilUpdatedAt` accuracy on PoolMember
- Fix `poolTotalAmountDistributedUntilUpdatedAt` on PoolMember
- Fix `totalNumberOfActivePools` on TokenStatistics
- Fix `totalSubscriptionsWithUnits` on TokenStatistics
- Use declarative eth calls for optimization
- Remove `receipt: true` from handlers, resulting in `gasUsed` staying null, but improving indexing performance
- Disable indexing of `Send` events. Use `Transfer` instead.
- Disable indexing of `TokenSnapshotLogs`

kasparkallas committed:
fix: adjustment flow rate & pool member total amount received

kasparkallas committed:
fix poolTotalAmountDistributedUntilUpdatedAt

kasparkallas committed:
fix totalNumberOfActivePools

kasparkallas committed:
fix totalSubscriptionsWithUnits

kasparkallas committed:
version bump & changelog

kasparkallas committed:
changelog

kasparkallas committed:
update deps

kasparkallas committed:
downgrade @graphprotocol/graph-cli

d10r committed:
updated flake.lock

hellwolf committed:
Merge branch 'dev' into 2025-03-10-subgraph-updates

hellwolf reviewed (approved):
echidna build on the head of nixpkgs is broken. I have a local fix, and the upstream merge is also nigh. I will do something here to fix soon.

kasparkallas committed:
use declarative eth calls

- it is a new subgraph feature
- we hope that it makes our subgraph index much faster

kasparkallas committed:
fix liquidation declared eth calls

kasparkallas committed:
dont update unnecessarily

kasparkallas committed:
add one more declared call

kasparkallas committed:
optimize creating less log entities

kasparkallas committed:
stop indexing send events

d10r committed:
fix cmd for graph network deployment

hellwolf commented:
Converting this to draft, and address conflicts once #2064 is merged.

kasparkallas committed:
merge

kasparkallas committed:
remove transaction receipts

kasparkallas committed:
fix changelog

mmd-afegbua committed:
upgrade local graph image (#2068)

kasparkallas committed:
Merge branch 'dev' into 2025-03-10-subgraph-updates

d10r commented:
Fails to sync on some networks.

Polygon:
```
Subgraph failed with non-deterministic error: failed to process trigger: block #11650651 (0xf772…05ea), transaction 387e2bb0974fd596ba92e8c9e159a60b7b2ea484603d360eee7bc7e5a67c1444: unknown param token, retry_delay_s: 201, attempt: 1, runner_index: 1576
```

Ethereum:
```
Subgraph failed with non-deterministic error: failed to process trigger: block #15874768 (0xf0d2…0dae), transaction e790ace5da74c61d11374a9d6ad63c0090ac6481910d21ecebca5398adb0ec0d: unknown param token, retry_delay_s: 487, attempt: 2, runner_index: 1579
```

Working on Base and Scroll, it may be related to some kind of incompatibility with older versions of the protocol which were never deployed on more recently added chains.

kasparkallas committed:
fix event param name

kasparkallas committed:
remove tokensnapshotlog indexing

mmd-afegbua committed:
continue on error for int tests

mmd-afegbua committed:
added login step for goldsky

mmd-afegbua committed:
added goldsky new api key

kasparkallas committed:
fix totalstreamedin not getting updated

kasparkallas committed:
Merge branch '2025-03-10-subgraph-updates' of https://github.com/superfluid-org/protocol-monorepo into 2025-03-10-subgraph-updates

kasparkallas committed:
Merge branch 'dev' into 2025-03-10-subgraph-updates

mmd-afegbua committed:
[SUBGRAPH] Local subgraph test fix (#2072)

* test increased mocha timeout
---------

Co-authored-by: Kaspar Kallas <git@kasparkallas.com>
Co-authored-by: Miao ZhiCheng <miao@superfluid.finance>

hellwolf committed:
Delete .github/workflows/handler.publish-pr-packages.yml

kasparkallas committed:
fix the bug with test

kasparkallas committed:
Merge branch '2025-03-10-subgraph-updates' of https://github.com/superfluid-org/protocol-monorepo into 2025-03-10-subgraph-updates

kasparkallas committed:
updated yarn.lock

hellwolf committed:
fix foundry commit

hellwolf committed:
Merge branch 'fix-foundry-monthly' into 2025-03-10-subgraph-updates

d10r committed:
changed subgraph deploy code for graph network, alchemy
